app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -55,6 +79,7 @@ def enough_resource_to_start_worker?(_worker_class)
   def cleanup_orphaned_worker_rows
     unless current_pods.empty?
       orphaned_rows = miq_workers.where.not(:system_uid => current_pods.keys)
+                                 .where.not(:status => MiqWorker::STATUSES_STARTING)
```

Comment:
just verifying that this should be an `AND`

Looking at it, it looks like it should but just want to check

Reply:
I added a comment above this to make the intention more clear